### PR TITLE
(ft-HelloWord): Displaying Hello word

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description

This Pull Request is about Creating Hello World Message

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Setting up an empty React Project With vite
- [x] Create component of Hello World message

## Description of tasks that were expected to be completed and that are not yet completed

- [x] none

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run `npm start`
3. step 3
4. step 4...

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable
Screenshot 2023-10-02 at 15 35 22

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by AristideI — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>React Exercices</title>
```
Comment (arjorb) [inline index.html:7]: Good job here, it's really a good practice to always change the title of the project 🙌🏽

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,10 @@
-import './App.css'
-import HelloWorld from './Components/HelloWorld'
-import UserClick from './Components/CapturingUserClick'
+import React from 'react'
```
Comment (arjorb) [inline src/App.jsx:1]: You no longer need to import React from "react". Starting from the release 17 of React, JSX is automatically transformed without using React. It simplifies the code by removing for unnecessary imports.

File: src/App.jsx [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
+import HelloWorld from './Components/HelloWorld/HelloWorld'
 function App() {
 
   return (
     <>
-    {/* <HelloWorld/> */}
-    <UserClick/>
+    <HelloWorld/>
     </>
```
Comment (arjorb) [inline src/App.jsx:8]: I think no need of this fragment tags as long as you are returning only one component! if it doesn't throws errors you can just move them! They are just adding the complexity of the codebase 🫣!

**Review by arjorb — COMMENTED** [review]
@kunda4, great job on these first exercises! I really liked your readme file, it is well structured (perfect 🙌🏽)! I can learn from it, haha. As for the codebase, it's really nice, but I left a few comments on things that you can change to improve the codebase.

**Pull request merged**